<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortWidgets</class>
 <widget class="QWidget" name="PortWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>361</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="ptn_refresh">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CmPortlist">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LblBaud">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>53</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CmbBaud">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CmbDataBits">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LblDataBits">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>53</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>数据位</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CmbStopBits">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LblStopBits">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>53</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>停止位</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CmbParity">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LblParity">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>53</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>校验位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptn_conn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭页面</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
